subsys/testsuite/ztest/Kconfig

```diff
@@ -35,23 +34,35 @@ config ZTEST_ASSERT_VERBOSE
 	  1 Write file, line number, function and reason for failed assertions
 	  2 Log also successful assertions
 
+config ZTEST_THREAD_PRIORITY
+	int "Testing thread priority"
+	default -1
+	help
+	  Set priority of the testing thread. Default is -1 (cooperative).
+
+config ZTEST_RETEST_IF_PASSED
+	bool "Reset the board to test again if the test passed"
+	select REBOOT
+	help
+	  If the test passed reset the board so it is run again.  This
+	  may be used as an alternative to manual resets when
+	  attempting to reproduce an intermittent failure.
+
+endif # ZTEST
+
 config ZTEST_MOCKING
 	bool "Mocking support functions"
-	depends on ZTEST
 	help
 	  Enable mocking support for Ztest. This allows the test to set
 	  return values and expected parameters to functions.
 
+if ZTEST_MOCKING
```

Comment:
Note that this `if` is redundant with the `depends on` below, as @ulfalizer tends to kindly reminds us.

Reply:
Bah.  Removed it in all the others, missed it here.  Thanks.

Reply:
Fixed.